vo_new/components/charts/MiniChart.tsx

```
@@ -0,0 +1,183 @@
+            type: 'scatter',
+            data: scatterData,
+            symbolSize: 4,
+            itemStyle: { color: colors[3] }
+          }]
+        };
+
+      case 'funnel':
+        const funnelData = data['x-axis'].map((name, index) => ({
+          name: String(name),
+          value: data['y-axis'][index] || 0
+        })).sort((a, b) => b.value - a.value);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)